proper/docker-proper.go

```diff
@@ -147,7 +147,8 @@ func removeImages(client *dockerclient.DockerClient, images []*dockerclient.Imag
 	for _, image := range images {
 		log.Print("rmi ", image.Id)
 		if !*dry {
-			if err := client.RemoveImage(image.Id); err != nil {
+			_, err := client.RemoveImage(image.Id)
+			if err != nil {
 				/*				if derr, ok := err.(dockerclient.Error); ok {
```

Comment:
I guess we need to include this again so it skips images that are still running?